quantum-viz/pyproject.toml

```diff
@@ -24,6 +24,7 @@ python = "^3.6.1"
 click = "^8.0.1"
 ipython = "<8.0.0"
 varname = "^0.8.1"
+qiskit-terra = "^0.18.3"
```

Comment:
We should probably move this to extras, however it would require some changes to install optional dependencies in the tests. If you want to take a stab at that feel free, if not I can help set that up, LMK